[2025-10-31T02:45:34.650+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-31T02:45:34.678+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_postgres_dag.extract_api_data manual__2025-10-31T02:45:33.315808+00:00 [queued]>
[2025-10-31T02:45:34.686+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_postgres_dag.extract_api_data manual__2025-10-31T02:45:33.315808+00:00 [queued]>
[2025-10-31T02:45:34.687+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-10-31T02:45:34.700+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_api_data> on 2025-10-31 02:45:33.315808+00:00
[2025-10-31T02:45:34.705+0000] {standard_task_runner.py:63} INFO - Started process 228 to run task
[2025-10-31T02:45:34.708+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'api_to_postgres_dag', 'extract_api_data', 'manual__2025-10-31T02:45:33.315808+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/api_to_postgres_dag.py', '--cfg-path', '/tmp/tmph679_qod']
[2025-10-31T02:45:34.711+0000] {standard_task_runner.py:91} INFO - Job 56: Subtask extract_api_data
[2025-10-31T02:45:34.771+0000] {task_command.py:426} INFO - Running <TaskInstance: api_to_postgres_dag.extract_api_data manual__2025-10-31T02:45:33.315808+00:00 [running]> on host 754154eb0275
[2025-10-31T02:45:34.854+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='lenny' AIRFLOW_CTX_DAG_ID='api_to_postgres_dag' AIRFLOW_CTX_TASK_ID='extract_api_data' AIRFLOW_CTX_EXECUTION_DATE='2025-10-31T02:45:33.315808+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-31T02:45:33.315808+00:00'
[2025-10-31T02:45:34.856+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-31T02:45:34.988+0000] {logging_mixin.py:188} INFO - âœ… Extracted and saved raw data.
[2025-10-31T02:45:34.990+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-31T02:45:34.991+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-31T02:45:35.002+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=api_to_postgres_dag, task_id=extract_api_data, run_id=manual__2025-10-31T02:45:33.315808+00:00, execution_date=20251031T024533, start_date=20251031T024534, end_date=20251031T024535
[2025-10-31T02:45:35.042+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-31T02:45:35.064+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-31T02:45:35.067+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
